src/fabric/src/fabric2_server.erl

```diff
@@ -42,9 +42,10 @@
 
 
 -include_lib("couch/include/couch_db.hrl").
+-include_lib("kernel/include/file.hrl").
 
-
--define(CLUSTER_FILE, "/usr/local/etc/foundationdb/fdb.cluster").
+-define(CLUSTER_FILE_MACOS, "/usr/local/etc/foundationdb/fdb.cluster").
+-define(CLUSTER_FILE_LINUX, "/etc/foundationdb/fdb.cluster").
```

Comment:
Wonder if we could add Windows to the list. Not sure how supported it is, but I found the default cluster file location in the Rust library https://docs.rs/foundationdb/0.5.0/src/foundationdb/lib.rs.html#171-174

Reply:
cc @wohali

Reply:
I wasn't planning on doing WIndows support until all of this stuff settled down... I only want to have to touch it **once**.

Are we really close to a 4.0 release now?

Reply:
I don't think a 4.0 release is imminent, but I can take a shot at extending this PR to do best-effort Windows support.